### PR TITLE
Custom matrix effect for orbweaver chroma keypad

### PR DESCRIPTION
**Pull request by schaeggi**
Added custom matrix effect for Razer Orbweaver chroma keypad. Tested and it seems to work fine.

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/keyboards.py [hunk 1]
```diff
@@ -275,9 +275,17 @@ class RazerOrbweaverChroma(_RazerDeviceBrightnessSuspend):
     USB_VID = 0x1532
     USB_PID = 0x0207
     DEDICATED_MACRO_KEYS = True
-    METHODS = ['get_device_type_keypad', 'set_breath_random_effect', 'set_breath_single_effect',
-               'set_breath_dual_effect', 'set_static_effect', 'set_spectrum_effect', 'set_wave_effect', 'keypad_get_profile_led_red', 'keypad_set_profile_led_red', 'keypad_get_profile_led_green',
-               'keypad_set_profile_led_green', 'keypad_get_profile_led_blue', 'keypad_set_profile_led_blue', 'get_macros', 'delete_macro', 'add_macro', 'keypad_get_mode_modifier', 'keypad_set_mode_modifier']
+    HAS_MATRIX = True
+    MATRIX_DIMS = [5, 23]
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/keyboards.py]: `[4, 22]` would be the correct values for your device I think

**Review by z3ntu — CHANGES_REQUESTED** [review]
See comment, LGTM otherwise

File: daemon/openrazer_daemon/hardware/keyboards.py (hunk 1 above)
Comment (schaeggi) [reply daemon/openrazer_daemon/hardware/keyboards.py]: That's also what I thought and tried first. But then I could not access the fourth row. No idea why. But I'll revalidate and be back. Maybe I messed something up then.

Comment (schaeggi) [reply daemon/openrazer_daemon/hardware/keyboards.py]: Using [4, 22] leads to the following error ([5, 23] works fine):
device.fx.advanced.matrix[4, 1] = (255, 255, 0)
  File "/usr/lib/python3/dist-packages/openrazer/client/fx.py", line 988, in __setitem__
    assert 0 <= key[0] < self._rows, "Row out of bounds"

Comment (z3ntu) [reply daemon/openrazer_daemon/hardware/keyboards.py]: Sorry my bad. For some reason the 0th row isn't used on this device it seems. Then the matrix_dims should be `[5, 22]`. Row `1` to Row `4` and Column `0` to Column `21` are being sent by Synapse

**Comment by z3ntu** [comment]
Which email should I use as author for your commit? Two commits have the dizzit email and two the github-noreply one?

**Comment by schaeggi** [comment]
the dizzit one is fine.

**Review by z3ntu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**